docs/sources/whatsnew/whats-new-in-v7-4.md

```diff
@@ -0,0 +1,170 @@
+
+### Variable support in alert notifications
+
+You can now provide detailed information to alert notification recipients by injecting alert query data into an alert notification. Labels that exist from the evaluation of the alert query can be used in the alert rule name and in the alert notification message fields. The alert label data is injected into the notification fields when the alert is in the alerting state. When there are multiple unique values for the same label, the values are comma-separated.
+
+![Variable support in alert notifications](/img/docs/v74/alert-variable-notifications.png)
+
+### Content security policy support
+
+We have added support for [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP), a layer of security that helps detect and mitigate certain types of attacks, including Cross Site Scripting (XSS) and data injection attacks.
+
+CSP support is disabled by default, to enable it you must set `content_security_policy = true` in the Grafana configuration. If enabling it, you should also review, and potentially tweak, the CSP header template, via the configuration setting `content_security_policy_template`.
+
+You can lock down what can be done in the frontend code. Lock down what can be loaded, what JavaScript is executed. Not compatible with some plugins.
+
+### Elasticsearch data source updates
+
+Grafana 7.4 includes the following enhancements
+
+- Added support for serial differencing pipeline aggregation.
+- Added support for moving function pipeline aggregation.
+- Added support to the terms aggregation for ordering by percentiles and extended stats.
+- Updated date histogram auto interval handling for alert queries.
+
+> **Note:** We have deprecated browser access mode. Iit will be removed in a future release.
+
+### Azure Monitor updates
+
+The Azure Monitor query type was renamed to Metrics and Azure Logs Analytics was renamed to Logs to match the service names in Azure and align the concepts with the rest of Grafana.
+
+[Azure Monitor]({{< relref "../datasources/azuremonitor.md" >}}) was updated to reflect this change.
+
+### MQL support added for Google Cloud Monitoring
+
+You can now use Monitoring Query Language (MQL) for querying time-series data. MQL provides an expressive, text-based interface to retrieve, filter, and manipulate time-series data.
+
+Unlike the visual query builder, MQL allows you to control the time range and period of output data, create new labels to aggregate data, compute the ratio of current values to past values, and so on.
+
+MQL uses a set of operations and functions. Operations are linked together using the common pipe mechanism, where the output of one operation becomes the input to the next. Linking operations makes it possible to build up complex queries incrementally.
+
+Once query type Metrics is selected in the Cloud Monitoring query editor, you can toggle between the editor modes for visual query builder and MQL.
+
```

Comment:
Can we add the image from the draft?

Reply:
If someone can give it to me in a non-Gdoc form. I have yet to figure out how to get an image out of there short of taking a screenshot.